src/Connection_Oracle.php

```diff
@@ -3,8 +3,8 @@
 namespace atk4\dsql;
 
 /**
- * @deprecated use Oracle\Connection instead
+ * @deprecated use Oracle\Driver instead
  */
-class Connection_Oracle extends Oracle\Connection
+class Connection_Oracle extends Oracle\Driver
```

Comment:
Is it better? I meant the to only refactor the handler to driver, driver term is usually used for `\Pdo` driver.

Reply:
I see... But it extend `Connection`, so using 2 terms for the same seems not good to me :)

Reply:
The mice are playing but let's see what the big cats @romaninsh and @DarkSide666 will say about all this :)